src/ssl.c

```diff
@@ -19268,21 +19271,14 @@ static const char* wolfSSL_internal_get_version(ProtocolVersion* version)
 
     if (version->major == SSLv3_MAJOR) {
         switch (version->minor) {
-        #ifndef NO_OLD_TLS
-            #ifdef WOLFSSL_ALLOW_SSLV3
```

Comment:
This is to make code size smaller.
Is there a need for the strings?

Reply:
The strings are needed to generate cipher descriptions. For example the cipher `EDH-RSA-DES-CBC3-SHA` (which has the TLS minor version `TLSv1_MINOR`) is built even if TLS 1 is not allowed. Getting the description of this cipher results in an unknown TLS version.